### PR TITLE
Add the 'REDASH_ENFORCE_CSRF' environment variable

### PR DESCRIPTION
**Pull request by lucydodo**
Related: https://github.com/getredash/redash/issues/7079

### CODE REVIEW

**Comment by lucydodo** [comment]
Preview available: https://deploy-preview-734--redashio.netlify.app/help/open-source/admin-guide/env-vars-settings/

File: src/pages/kb/open-source/admin-guide/env-vars-settings.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ The follow is a list of settings and what they control:
...
 | `REDASH_CORS_ACCESS_CONTROL_ALLOW_HEADERS`      |                                                                                                                                                                                                                                                                    | Content-Type                                                                                                                               |
+| `REDASH_ENFORCE_CSRF`                           | Enforce CSRF token validation on API requests. **This is turned off by default to avoid breaking any existing deployments. but it is highly recommended to turn this toggle on to prevent CSRF attacks.**                                                          | false                                                                                                                                      |
```
Comment (justinclift) [inline src/pages/kb/open-source/admin-guide/env-vars-settings.md:68]: `... deployments. but is highly ...`

The full stop after "deployments" should be a comma instead. :wink:

Comment (lucydodo) [reply src/pages/kb/open-source/admin-guide/env-vars-settings.md]: Good point, I changed. :)

**Review by justinclift — APPROVED** [review]
Awesome, looks good. :smile:

**Comment by lucydodo** [comment]
Thanks for the review.

**Pull request merged**